src/components/PermissionPanel/index.tsx

```diff
@@ -0,0 +1,273 @@
+      },
+    }
+  );
+
+  const selectedRole = watch("role");
+  const permissionsValue = watch("permissions");
+  const notificationsValue = watch("notifications");
+  const roleRef = useRef<UserRole>(selectedRole);
+
+  const permissionColumns = useMemo<ColumnizedPBACGroups<AuthPermissions>>(() => {
+    if (!data?.retrievePBACDefaults && loading) {
+      return [];
+    }
+
+    const defaults = data?.retrievePBACDefaults?.find((pbac) => pbac.role === selectedRole);
+    if (!defaults || !defaults?.permissions) {
+      Logger.error("Role not found in PBAC defaults", { role: selectedRole, data });
+      return [];
+    }
+
+    const remappedPermissions: PBACDefault<AuthPermissions>[] = cloneDeep(defaults.permissions).map(
+      (p) => ({ ...p, checked: permissionsValue.includes(p._id) })
+    );
+
+    return columnizePBACGroups(remappedPermissions, 3);
+  }, [data, permissionsValue]);
+
+  const notificationColumns = useMemo<ColumnizedPBACGroups<AuthNotifications>>(() => {
+    if (!data?.retrievePBACDefaults && loading) {
+      return [];
+    }
+
+    const defaults = data?.retrievePBACDefaults?.find((pbac) => pbac.role === selectedRole);
+    if (!defaults || !defaults?.notifications) {
+      Logger.error("Role not found in PBAC defaults", { role: selectedRole, data });
+      return [];
+    }
+
+    const remappedNotifications: PBACDefault<AuthNotifications>[] = cloneDeep(
+      defaults.notifications
+    ).map((n) => ({ ...n, checked: notificationsValue.includes(n._id) }));
+
+    return columnizePBACGroups(remappedNotifications, 3);
+  }, [data, notificationsValue]);
+
+  const handlePermissionChange = (_id: AuthPermissions) => {
+    if (permissionsValue.includes(_id)) {
+      setValue(
+        "permissions",
+        permissionsValue.filter((p) => p !== _id)
+      );
+    } else {
+      setValue("permissions", [...permissionsValue, _id]);
+    }
+  };
+
+  const handleNotificationChange = (_id: AuthNotifications) => {
+    if (notificationsValue.includes(_id)) {
+      setValue(
+        "notifications",
+        notificationsValue.filter((n) => n !== _id)
+      );
+    } else {
+      setValue("notifications", [...notificationsValue, _id]);
+    }
+  };
+
+  const handleRoleChange = (selectedRole: UserRole) => {
+    if (selectedRole === roleRef.current) {
+      return;
+    }
+
+    const defaults = data?.retrievePBACDefaults?.find((pbac) => pbac.role === selectedRole);
+    setValue("permissions", defaults?.permissions?.filter((p) => p.checked).map((p) => p._id));
+    setValue("notifications", defaults?.notifications?.filter((n) => n.checked).map((n) => n._id));
+    roleRef.current = selectedRole;
+  };
+
+  useEffect(() => {
+    handleRoleChange(selectedRole);
+  }, [selectedRole]);
+
+  return (
+    <StyledBox>
+      <StyledAccordion elevation={0} data-testid="permissions-accordion">
+        <StyledAccordionSummary expandIcon={<ArrowDropDown />}>
```

Comment:
Just a suggestion, I think MUI has an icon that is closer to the design. Feel free to switch if you'd like.
`import ExpandMoreIcon from "@mui/icons-material/ExpandMore";`

Reply:
Agreed. Updated with the color fix 4553245.